scripts/build-sitemap.js

```diff
@@ -58,24 +57,23 @@ const addPath = (sitemap, url) => {
         sitemap = addPath(sitemap, `/map/${map.key}`);
     }
 
-    for(const itemType of itemTypes){
-        sitemap = addPath(sitemap, `/items/${itemType.key}`);
+    const itemTypes = await got('https://api.tarkov.dev/graphql?query={%20itemCategories{%20normalizedName%20}%20}', {
+        responseType: 'json',
+    });
+
+    for (const itemType of itemTypes.body.data.itemCategories) {
+        sitemap = addPath(sitemap, `/items/${itemType.normalizedName}`);
     }
 
-    const allItems = await got('https://api.tarkov.dev/graphql?query={%20itemsByType(type:%20any){%20normalizedName%20}%20}', {
+    const allItems = await got('https://api.tarkov.dev/graphql?query={%20items{%20normalizedName%20}%20}', {
```

Comment:
This doesn't need to be done this **this** PR but in a future PR we should refactor this to use a request body that is well formatted rather than query strings that are URL encoded. It will make it easier to maintain in the future

Reply:
Feel free!